### ISSUE TITLE
Setup translations pipeline w/ Coronatracker to support other language-> High Priority Dutch

### ISSUE THREAD

**Issue by tesla809** [leather-io/extension#355]
Coronatracker (a tool to help monitor COVID19 symptoms) has its first real-life user, a doctor in Belgium. However, his patients are older and speak Dutch. 

The deadline to deliver the MVP to the doctor is Monday, May 11th, 2020. Failure to do so will endanger any sort of institutional partnership, which are VERY hard to gain in the medical field.

Blockstack Connect is written in English, which poses a major usability problem for non-English speakers.

The CoronaTracker team has a translations team working on translations in Dutch, Spanish, Hindi, Chinese Mandarin, Arabic, Russian, German, Slovak, French, Italian and more. 

We wish to create a pipeline to funnel translations into the UI so that Blockstack Connects language barrier is not a hindrance to our user acquisition funnel. 

Here is our associated issue on our repo:
https://github.com/COVID-19-electronic-health-system/Corona-tracker/issues/678

**Comment by tesla809** [comment]
A low-cost solution could be to use:
 
https://crowdin.com/ 

as a way to handle translations easier.

OpenCollective uses it to translate their website and seems to work well. I have used it to help with translations.

**Comment by tesla809** [comment]
See our implementation:
https://github.com/COVID-19-electronic-health-system/Corona-tracker/blob/master/client/src/i18n.js

**Comment by agraebe** [comment]
I have been facilitating the communications here a bit and wanted to update directly on the ticket. The expectation should be that we (as PBC) can't deliver this before May 11th. Therefore, the solution we identified (together with @tesla809 and @hstove) was to facilitate and [provide guidance](https://paper.dropbox.com/doc/Internationalization-of-Blockstack-Connect--AzJxmw96oNrFcNLJGs_goEUxAQ-hmh05r7p5eBf3CLTJyFvF) on how to enable i18n in the connect and the authenticator project respectively. With that, you could deploy your own version and hopefully contribute with a PR to make i18n possible for everyone.

cc @markmhx @ryanarndtcm

**Comment by AdhamAH** [comment]
@agraebe I followed the instructions [here](https://paper.dropbox.com/doc/Internationalization-of-Blockstack-Connect-hmh05r7p5eBf3CLTJyFvF) that you provided
But when I run `yarn lerna run dev --stream --scope app` I have an error
```bash
lerna notice cli v3.20.2
lerna info versioning independent
lerna notice filter including "app"
lerna info filter [ 'app' ]
lerna ERR! EFILTER No packages remain after filtering [ 'app' ]
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

**Comment by agraebe** [comment]
@hstove is probably the best person to help

**Comment by hstove** [comment]
@AdhamAH we recently renamed that package. Now it's referred to as `@blockstack/app`. So the new command is: `yarn lerna run dev --stream --scope @blockstack/app`

**Comment by AdhamAH** [comment]
I am not near my laptop.
I actually tried that. And after linking our app `yarn link @blockstack/connect` I got an error that there are files missing ending ` compiled.js` if I remember right. I run the command to compile and I got an invalid hook call somewhere in the main `app` folder, that's when I came here 
So could you please if you have the time to check running that? 
Also in couple hours I will be near my computer and have the time to try again and report back.
Thanks for the help!👍🏼 @hstove

**Comment by hstove** [comment]
If you see the 'invalid hook error' in your app, then that's probably because you have different versions of React in your app and in Connect. You can probably solve this by using yarn resolutions, and there are other workarounds that you can follow by clicking the link in the "Invalid hook" error message.

Once you're back at a computer, do you think you could post some very specific reproduction steps here? I.e. link to your app's Github, then tell me what steps you're following, and then when you see the error. I should be able to be more helpful with that info.

Thanks!

**Comment by AdhamAH** [comment]
I am working with this app [CoronaTracker](https://github.com/COVID-19-electronic-health-system/Corona-tracker) 
I pulled the latest master from `blockstack/ux`. After I run `yarn start` at `packages/connect` folder I got this error 
![Screenshot 2020-06-17 at 20 42 02](https://user-images.githubusercontent.com/55054963/84997356-c641fc00-b14e-11ea-9320-30652f4350ee.png)
After searching and trying several things, it worked after performing a `yarn upgrade` 
Now, I am trying to see how to apply `i18next`. `blockstack/ux` is SSR right? And this is different from what we have in our app, we need to use [this](https://react.i18next.com/latest/i18nextprovider). Any thoughts? 
I will work more on it and see
***P.S: I am new and not good with terminology or asking questions  😅***
